### PR TITLE
network: miscellaneous changes

### PR DESCRIPTION
**Pull request by thiagowfx**
See commit history for changes.

Bug: https://github.com/GoogleChromeLabs/chromium-bidi/issues/765
Spec: https://w3c.github.io/webdriver-bidi/#module-network

### CODE REVIEW

**Comment by thiagowfx** [comment]
Will generate WPT expectations.

File: src/bidiMapper/domains/network/networkRequest.ts [hunk 1]
```diff
@@ -29,54 +28,76 @@ import {Network} from '../../../protocol/protocol.js';
...
-  #responseReceivedExtraInfoEvent:
-    | Protocol.Network.ResponseReceivedExtraInfoEvent
-    | undefined;
+  #requestWillBeSentEvent?: Protocol.Network.RequestWillBeSentEvent;
+  #requestWillBeSentExtraInfoEvent?: Protocol.Network.RequestWillBeSentExtraInfoEvent;
+  #responseReceivedEvent?: Protocol.Network.ResponseReceivedEvent;
+  #responseReceivedExtraInfoEvent?: Protocol.Network.ResponseReceivedExtraInfoEvent;
 
-  #beforeRequestSentDeferred: Deferred<void>;
+  #requestWillBeSentDeferred: Deferred<void>;
```
Comment (Lightning00Blade) [inline src/bidiMapper/domains/network/networkRequest.ts:51]: How I viewed it is that this is that Deferred are for the output BiDi commands, this one being `network.beforeRequestSent`

Comment (thiagowfx) [reply src/bidiMapper/domains/network/networkRequest.ts:51]: OK, reverted

File: wpt-metadata/chromedriver/headless/webdriver/tests/bidi/network/response_started/response_started_cached.py.ini [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [response_started_cached.py]
+  expected: TIMEOUT
```
Comment (sadym-chromium) [inline wpt-metadata/chromedriver/headless/webdriver/tests/bidi/network/response_started/response_started_cached.py.ini:2]: Do you know why the tests started timing out?

**Review by sadym-chromium — APPROVED** [review]

File: wpt-metadata/chromedriver/headless/webdriver/tests/bidi/network/response_started/response_started_cached.py.ini (hunk 2 above)
Comment (thiagowfx) [reply wpt-metadata/chromedriver/headless/webdriver/tests/bidi/network/response_started/response_started_cached.py.ini:2]: Time out is better than the status quo: 
image

But no, I don't know

**Pull request merged**